source/appModules/miranda32.py

```diff
@@ -1,6 +1,7 @@
+# -*- coding: UTF-8 -*-
 #appModules/miranda32.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2012 NVDA Contributors
+#Copyright (C) 2006-2019 NV Access Limited, Aleksey Sadovoy, Peter Vágner, Joseph Lee, Bill Dengler

```

Comment:
Just curious, but how did you work out that Aleksey Sadovoy, Peter Vágner and Joseph Lee contributed to this file? Did you look at the repository history?